### DISCUSSION TITLE
Is it possible to unarchive repository one it is archived?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#23080] [New to GitHub]
<p>The GitHub documentation on <a href="https://docs.github.com/en/repositories/archiving-a-github-repository/archiving-repositories" rel="noopener nofollow ugc">archiving repositories</a> does not mention if this is a <strong>one way operation</strong> or if let say I did archive the wrong repository my mistake or if I change my mind I can “un-archive” it and make is active again?</p>

**Comment by jsoref** [comment, marked as answer]
<p>It’s reachable via <code>/settings</code> in the same place as you would have archived the repository:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/a/1a39c6f073e34ba26a84d24f77d75b3234617302.png" data-download-href="/uploads/short-url/3K0ct4uVHZfuNTJInkGqB3qShNg.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/2119212/181074719-615ee5f1-0af2-40a1-a903-f2f1f3237245.png" alt="image" data-base62-sha1="3K0ct4uVHZfuNTJInkGqB3qShNg" width="690" height="335" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/a/1a39c6f073e34ba26a84d24f77d75b3234617302_2_690x335.png, https://aws1.discourse-cdn.com/github/original/3X/1/a/1a39c6f073e34ba26a84d24f77d75b3234617302.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/1/a/1a39c6f073e34ba26a84d24f77d75b3234617302.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/a/1a39c6f073e34ba26a84d24f77d75b3234617302_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">796×387 37.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>Interesting that you mention a lack of documentation, there is documentation for restoring deleted repositories (you apparently have 90 days for isolated repositories – you can’t recover deleted forks in live networks – although you could probably retrieve the individual commits):<br>
<a href="https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository" class="onebox" target="_blank" rel="noopener">https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository</a><br>
And this is linked from its sibling:<br>
<a href="https://docs.github.com/en/repositories/creating-and-managing-repositories/deleting-a-repository" class="onebox" target="_blank" rel="noopener">https://docs.github.com/en/repositories/creating-and-managing-repositories/deleting-a-repository</a></p>

**Comment by jsoref** [comment]
<p>Ticket to document: <a href="https://github.com/github/docs/issues/18753" class="inline-onebox">Create documentation for unarchiving-repositories · Issue #18753 · github/docs · GitHub</a></p>

**Reply by jsoref** [reply]
For the record, this is now documented: https://docs.github.com/en/repositories/archiving-a-github-repository/archiving-repositories#unarchiving-a-repository

**Comment by queenofcorgis** [comment]
<p><a class="mention" href="/emil-yankov_rimes">@emil-yankov_rimes</a> :wave:t3: thanks for posting - was <a class="mention" href="/jsoref">@jsoref</a> able to assist, if so, don’t forget to mark their comment as the solution, thanks!</p>

**Comment by th-yoon** [comment]
For those who need to programmatically unarchive a batch of multiple repositories, I implemented one based on python using graphql api.
Check this out if you're interested: https://pypi.org/project/sunrise-repos/
Source code: https://github.com/th-yoon/sunrise-repos

**Comment by bcsgh** [comment]
Having "Unarchive" in the danger-zone seems strange as it's not something that can't be un-done. Maybe there needs to be "warning zone"?
